src/ar/ARTester.cpp

```diff
@@ -91,28 +104,38 @@ int main(int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 	cam_config[cam::KEY_INTRINSIC_PARAMS] >> PARAMS;
+	// read filename or camera ID, and open camera.
+	if (!cam_config[cam::KEY_FILENAME].empty() && !parser.has("fo"))
+	{
+		cam_file = (std::string)cam_config[cam::KEY_FILENAME];
+	}
+	else if (!cam_config[cam::KEY_CAMERA_ID].empty() && !parser.has("co"))
+	{
+		cam_id = (int)cam_config[cam::KEY_CAMERA_ID];
+	}
```

Comment:
We're basically just reimplementing all the camera config file parsing that's already implemented in the `Camera` class here, which isn't great, so we'll definitely need to refactor this later